sky/clouds/azure.py

```diff
@@ -177,16 +206,42 @@ def get_image_size(cls, image_id: str, region: Optional[str]) -> float:
         return size_in_gb
 
     @classmethod
-    def get_default_instance_type(
-            cls,
-            cpus: Optional[str] = None,
-            memory: Optional[str] = None,
-            disk_tier: Optional[resources_utils.DiskTier] = None
-    ) -> Optional[str]:
-        return service_catalog.get_default_instance_type(cpus=cpus,
-                                                         memory=memory,
-                                                         disk_tier=disk_tier,
-                                                         clouds='azure')
+    def get_community_image(cls, image_id, region,
+                            compute_client) -> CommunityGalleryImage:
+        # Community gallery image id format:
+        # /CommunityGalleries/<gallery-name>/Images/<image-name>
```

Comment:
Do we introduce this new format for people to specify the image id from community gallery? Is there a refer link for this format specifically?

Reply:
This is the public resource ID for a community image, you can find it here: https://portal.azure.com/?feature.msaljs=true#@adminassemblesys.onmicrosoft.com/resource/providers/Microsoft.Compute/locations/westus2/CommunityGalleries/SkyPilot-fe1926f1-7aff-4300-90d6-51fe7a190de7/Images/skypilot-cpu/overview